os/net/net_initialize_netmgr.c

```diff
@@ -99,6 +100,10 @@ void net_setup(void)
 
 void net_initialize(void)
 {
+	/* Initialize the locking facility */
+
+	net_lockinitialize();
```

Comment:
netmgr doesn't use net_lock, so remove this

Reply:
The UDS needs net_lock. When should it be initialized?